models/news.js

```diff
@@ -55,4 +55,22 @@ exports.selectCommentsByArticleId = (articleId) => {
     .then((comments) => {
         return comments.rows
     })
+}
+
+exports.editArticle = (articleId,articleInfo) => {
+    if(!Number(articleId)){
+        return Promise.reject({status:400, msg:'id is not a number'})
+    }
+    const {inc_votes} = articleInfo
+    return checkArticleExists(articleId).then(() => {
```

Comment:
There isn't actually a need to check if the article exists here - it would be more efficient to attempt the update and check if any rows have been affected afterwards - take a look on the result object given t us by `db.query` - iot may have something interesting to use 👀